src/components/molecules/ChatList/ChatList.tsx

```diff
@@ -24,29 +16,20 @@ import "./ChatList.scss";
 import { ChatMessage } from "./ChatMessage";
 
 interface ChatListProps {
+  usersById: Record<string, User>;
   messages: WithId<RestrictedChatMessage | PrivateChatMessage>[];
   emptyListMessage?: string;
   allowDelete?: boolean;
   deleteMessage: (id: string) => Promise<void>;
 }
 
 const ChatList: React.FC<ChatListProps> = ({
+  usersById,
   messages,
   allowDelete,
   emptyListMessage,
   deleteMessage,
 }) => {
-  const usersById = useSelector(venueChatUsersSelector) ?? {};
```

Comment:
just wanting to check - did this move to the parent as a DRY up?

Reply:
- `src/components/molecules/Chatbox/Chatbox.tsx` receives `usersById` as a prop, and passes it to this component
- `ChatBox` is embedded by
  - `src/components/molecules/ChatsList/ChatsList.tsx`
  - `src/components/molecules/VenueChat/VenueChat.tsx`
  - `src/components/organisms/PrivateChatModal/PrivateChatModal.tsx`
  - `src/components/organisms/UserProfileModal/UserProfileModal.tsx`
  - `src/pages/FriendShipPage/FriendShipPage.tsx`
- I can see changes in this PR to `VenueChat`/`ChatsList` to wire this up, but I don't think I see changes for `PrivateChatModal`, `UserProfileModal`, or `FriendShipPage`
  - Do these extra components need to be wired up as well?

Reply:
I need to pass the users from the parent component because the chatlist is using a different set of users in every component it's being used. In venue chat I need the users from the venue chat, in private chat I need the users from the private chats and etc...

Reply:
No prob, that’s what I suspected! Thanks!

Reply:
from https://github.com/sparkletown/sparkle/pull/845#discussion_r533060123

> Do these extra components need to be wired up as well?